src/test/ui/async-await/in-trait/fn-not-async-err2-rpitit.rs

```diff
@@ -0,0 +1,22 @@
+// check-pass
```

Comment:
Is this identical to src/test/ui/async-await/in-trait/async-example-desugared.rs? If so, plz delete.

Reply:
oh yeah, it's the same. I'll get rid of it